src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $white-color;
+  padding-top: 32px;
+  font-size: 10px;
+
+  transition: all 300ms linear;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-inline: 19px;
+  }
+
+  &__content {
+    padding: 40px 16px 16px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+
+    transition: all 300ms linear;
+  }
+
+  &__code {
+    color: $secondary-color;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      color: $secondary-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-count {
+      color: $main-color;
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    text-align: center;
+    max-height: 40px;
+
+    &-link {
+      display: block;
+      padding-block: 12px;
+      text-decoration: none;
+      text-transform: uppercase;
+
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      color: #fff;
+
+      border: 1px solid $blue-color;
+      border-radius: 5px;
+      background-color: $blue-color;
+
+      transition: all 300ms linear;
+
+      &:hover {
+        color: $blue-color;
+        background-color: #fff;
+      }
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: $hover-title-color;
```

Comment:
The color is set to `$hover-title-color`. Ensure that `$hover-title-color` is defined in your variables file. If it's not defined, this will cause an error.